entry_types/scrolled/app/controllers/pageflow_scrolled/editor/chapters_controller.rb

```diff
@@ -0,0 +1,48 @@
+module PageflowScrolled
+  module Editor
+    # @api private
+    class ChaptersController < ActionController::Base
+      include Pageflow::EditorController
+      skip_before_action :verify_edit_lock, only: :show
+      respond_to :json
+
+      def create
+        entry = Pageflow::DraftEntry.find(params[:entry_id])
+        chapter = Chapter.create(chapter_params.merge(revision: entry.draft))
+
+        render json: chapter, status: :created
+      end
+
+      def update
+        chapter = Chapter.find(params[:id])
+        chapter.update_attributes(chapter_params)
+
+        render json: chapter
+      end
+
+      def destroy
+        chapter = Chapter.find(params[:id])
+        chapter.destroy
+
+        render json: chapter
+      end
+
+      def order
+        entry = Pageflow::DraftEntry.find(params[:entry_id])
+        storyline = Storyline.all_for_revision(entry.draft).first
+
+        params.require(:ids).each_with_index do |id, index|
+          storyline.chapters.update(id, position: index)
```

Comment:
The `order` actions of the other controllers already allow moving content elements/sections between arbitrary sections/chapters within the storyline. Let's also allow moving chapters between storylines here and extend the other `order` actions to also allow moving content elements/sections between storylines.